lib/health/exercise.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Health.Exercise do
+  @moduledoc """
+  An Exercise
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "exercises" do
+    field :name, :string
+    field :description, :string
+    field :difficulty, :integer
+
+    timestamps()
+  end
+
+  @doc false
+  @spec changeset(struct(), map()) :: %Ecto.Changeset{}
+  def changeset(exercise, attrs) do
+    exercise
+    |> cast(attrs, [:name, :description, :difficulty])
+    |> validate_required([:name, :description, :difficulty])
+    |> validate_number(:difficulty, greater_than: 0, less_than: 11)
+  end
+
+  @spec all :: [%Health.Exercise{}]
+  def all do
+    Health.Exercise |> Health.Repo.all
```

Comment:
You can alias these up top with `Health.{Exercise, Repo}`

This has the super awesome side effect of always being able to look at the top and see what the module is coupled to... once there are too many nested aliases it is time to break something out!

Reply:
I've been intentionally leaving them explicit until I get the hang of what I'm calling. Avoiding the alias. :)

I like your point of the super awesome side effect!